app/views/posts/show.html.erb

```diff
@@ -1,3 +1,16 @@
-<div>
-    <h1>This is the show for posts</h1>
-</div>
\ No newline at end of file
+<main>
+    <div class="detail">
+        <div class="user">
+            <h4>Post #<%= @post.id %> by <%= @post.author.name %></h4>
+            <p>Comments:<%= @post.comments.count %>, Likes: <%= @post.likes.count %></p>
```

Comment:
- [ ] Well done for creating the `create` action in the `LikesController` 👏 . However, I am unable to like a post. Please implement a way to like a post when the like button or link is clicked 👍